drafts/use-cases/3.obtaining-events.md

```diff
@@ -0,0 +1,110 @@
+## Entry point
+
+### Story
+As an application user
+I want to display all my events
+So I can find out what awaits me in the nearest future.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api/events");
+for (var member of data.members) {
+    // do something with the _member_, i.e. display it
+}
+```
+
+### Details
+My application used a discovered either from the entry-point or the API documentation URL to my events.
+It sends a GET request to that URL and fetches the events:
+
+```
+GET /api/events
+
+HTTP 200 OK
+```
+```javascript
+{
+    "@id": "/api/events",
+    "totalItems": 1,
+    "members": [
+        {
+            "@id": "/api/events/1",
+            "eventName": "Event 1",
+            "eventDescription": "Some event 1",
+            "startDate": "2017-04-19",
+            "endDate": "2017-04-19"
+        }
+    ]
+}
+```
+
+It is possible now to display one event in the calendar.
+
+### Considerations
+
+#### Some issues
```

Comment:
@RubenVerborgh thoughts on the issues below?

My gut tells me that knowing in advance isn't really that important. We may need that information in actual (Partial)Collection's representation. Is it already specified how we'd do that?

Similarly with whether the collection is partial or not should not be of utmost importance. 

In any case let's reflect back on https://www.w3.org/community/hydra/wiki/Pagination#PartialCollectionView and https://www.w3.org/community/hydra/wiki/Collection_Interaction_Flow for earlier ideas and maybe move that to a separate document for discussion.